telegram/_bot.py

```diff
@@ -142,7 +142,12 @@ class Bot(TelegramObject, AbstractAsyncContextManager):
 
     .. seealso:: :attr:`telegram.ext.Application.bot`,
         :attr:`telegram.ext.CallbackContext.bot`,
-        :attr:`telegram.ext.Updater.bot`
+        :attr:`telegram.ext.Updater.bot`,
+        `Builder Pattern <https://github.com/\
+        python-telegram-bot/python-telegram-bot/wiki/Builder-Pattern>`_,
+        `Making Your Bot Persistent <https://github.com/\
+        python-telegram-bot/python-telegram-bot/wiki/Making-your-bot-persistent>`_
```

Comment:
IMO we can skip this. Handling bot instances is one thing that one needs to be concerned with when using persistence, but using this class doesn't really make you interested in persistence - storing data does :)

Reply:
Done